packager/packager.cc

```diff
@@ -1298,6 +1289,9 @@ vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers
                     auto &file = job.file.data(gs);
                     if (file.sourceType == core::File::Type::Normal) {
                         job = rewritePackagedFile(gs, move(job));
```

Comment:
For symmetry and if you want, you could hoist the part where `rewritePackagedFile` creates a `core::Context` out to the call site here.

That would drive home the point that both these two functions (`rewritePackagedFile` and `rewritePackage`) operate on immutable global states.

Reply:
Fixed